lambda_functions/admin_service/update_item.py

```diff
@@ -68,6 +69,11 @@ def update_item(event, context, is_test=False, session=None):
             return response_cors
 
     item = update_object(item, is_test, session)
+
+    if(item.status == 'rejected'):
+        EventPublisher().publish_event(
```

Comment:
Ich kenne jetzt Python zu wenig. Aber sollte der EventPublisher nicht eher eine singleton Klasse sein. 
Also das alle denselben EventPublisher verwenden.

Oder alternativ eine static function.

Aber wie gesagt, ich bin nicht tief genug drin, eventuell ist es auch Blödsinn was ich hier schreibe ;-)

Reply:
Sollte in dem Fall glaube ich tatsächlich relativ egal sein.
Der Code wird in der Lambda Funktion ohnehin nur einmal durchlaufen.

Hatte das ganze nur in eine Klasse verpackt, falls tatsächlich mal der Fall eintritt, dass in einer Lambda mehrere Events gepublished werden müssen.